src/scene/vcModel.cpp

```diff
@@ -439,7 +439,11 @@ void vcModel::HandleSceneExplorerUI(vcState *pProgramState, size_t * /*pItemID*/
   ImGui::InputScalarN(vcString::Get("sceneModelScale"), ImGuiDataType_Double, &scale.x, 1);
 
   if (ImGui::IsItemDeactivatedAfterEdit())
+  {
     repackMatrix = true;
+    if (scale.x == 0.0f)
+      scale.x = 1e-7; // Simple Hack to avoid 0 scale
```

Comment:
This could probably have just been `scale.x = udMax(scale.x, 1e-7);`, but I think this makes it clearer the issue is specifically 0, even without this comment.

Reply:
Fixed